omnibus/tests/verify-install.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+set -e
+
+# Ensure user variables are set in git config
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+
+
+# These environment vars were set by default
+export CHEF_LICENSE="accept-no-persist"
+export PATH=/opt/chef-workstation/bin:/opt/chef-workstation/embedded/bin:$PATH
+# Ensure no errors when running `chef` out of Workstation
+
+
+export_gem_location_for() {
+
+  gem_which=$(gem which "$1")
+  gem_dir=$(dirname "$gem_which")/..
+  gem_location=$(realpath "$gem_dir")
+  export gem_location
+}
+export_temp_dir() {
+  temp_dir="$(mktemp -d -t)"
+  export temp_dir
+}
+
+cleanup_temp_dir() {
+  if [ -d "$temp_dir" ]; then
+    rm -rf "$temp_dir"
+  fi
+  unset temp_dir
+}
+
+
+# Ensure our temporary directory is deleted
+# given function cleanup(), then we can:
+# Note: the interrupt is handled in a running ruby app as a failure,
+#       and is not picked up by the trap
+trap cleanup_temp_dir EXIT
+trap 'echo "Interrupt received, stopping"; cleanup_temp_dir; exit 127' INT TERM
+
+# Load the WS gem environment so that we can reach the gems we care about
+# Previously these actions would run from within a `chef` command,
+# where the ruby environment is already set up
+echo "Preparing environment"
+eval "$(chef shell-init bash)"
+
+echo 'Verifying package installation'
+
+echo "berks "
+berks -v
+echo "chef "
+chef -v
+echo "chef-client"
+chef-client -v
+echo "chef-solo"
+chef-solo -v
+echo "delivery-cli"
+delivery -V
+
+# In `knife`, `knife -v` follows a different code path that skips
+# command/plugin loading; `knife -h` loads commands and plugins, but
+# it exits with code 1, which is the same as a load error. Running
+# `knife exec` forces command loading to happen and this command
+# exits 0, which runs most of the code.
+#
+# See also: https://github.com/chef/chef-dk/issues/227
+echo "knife with commands and plugins loaded"
+knife exec -E true
+
+echo "kitchen"
+export_temp_dir
+pushd "$temp_dir"
+  # Kitchen makes a .kitchen even for -v
+  kitchen -v
+popd
+cleanup_temp_dir
+
+echo "ohai"
+ohai -v
+echo "foodcritic"
+foodcritic -V
+echo "inspec"
+inspec version
+
+
+echo "Verifying berkshelf"
+export_gem_location_for "berkshelf"
+pushd "$gem_location"
+  bundle install --quiet --with=development
```

Comment:
i don't think you need the `--with` because of how appbundler renders the Gemfile.  i don't think you get any choices here about with or without at all.